Source/UIApplication+openURL.h

```diff
@@ -0,0 +1,35 @@
+/*! @file UIApplication+openURL.m
```

Comment:
I wonder if it would be better to split these four substantive changes out into a separate PR?

Reply:
So you want a PR for adding the extensions to examples and another one which fix the compile error?

Reply:
Well, I think this is a valuable change, and that we should definitely address the use case. My concern is that the size of it may prevent a timely review. The “real” change here is only the UIApplication+openURL category, and the two files that make use of it. I totally agree that we should add examples - but I’m not sure we need to deal with that in the same change. Is splitting it something you’d be willing to do?

@WilliamDenniss WDYT?